mease_lab_to_nwb/convert_syntalos/convert_syntalos.py

```diff
@@ -1,27 +1,52 @@
 """Authors: Alessio Buccino, Cody Baker, Szonja Weigl, and Ben Dichter."""
 from pathlib import Path
+from isodate import duration_isoformat
+from datetime import timedelta
 
-from .syntalosnwbconverter import SyntalosNWBConverter
+from mease_lab_to_nwb import SyntalosNWBConverter
```

Comment:
use relative import

Reply:
The notebook uses from mease_lab_to_nwb, shall I change it there as well?

Reply:
?

Reply:
The processing pipeline for Syntalos, uses `from mease_lab_to_nwb import SyntalosRecordingExtractor`. The ReadMe also specifies installing it as a package, so they should have one called `mease_lab_to_nwb`.

But if the primary script is to use a relative import, should the processing pipeline as well?

Also, the tank-lab-to-nwb uses `tank_lab_to_nwb` for the external script, but relative imports on other components. Should that be made relative as well?

Reply:
All imports from within the same repo should be relative except for tests, which should always use absolute imports, and example code, like in a README file.